### PR TITLE
Issue 56

### PR DESCRIPTION
**Pull request by ealderman-uml**
Add instruction to check course dashboard after submission to setup assignment.

Added a section to the setup assignment instructing students to check the
course dashboard to confirm their submission was received
after they email their patches.

Addresses issue #56

### CODE REVIEW

File: assignments/setup.md [hunk 1]
```diff
@@ -277,6 +277,13 @@ Press:
 You should see your patch series including your introduction in the list of messages.
 If you do, congratulations! You successfully completed the setup.
```
Comment (charliemirabile) [inline assignments/setup.md:279]: This info is actually incorrect, the email will not show up, it only appears on dashboard (until after the deadline). I should have been clear about this problem in the related issue, but in connection with adding these correct instructions about checking the dashboard, it makes sense to remove this outdated info too.

**Review by charliemirabile — CHANGES_REQUESTED** [review]
This one has specific alterations I would like to see, outlined in my other comment but it along with all the other PRs you made are targeting the master branch when they should be targeting 2024_fall_uml. I sent you info about changing that on matrix, but just replying here too in case you aren't looking there.

**Comment by ealderman-uml** [comment]
- Rebased my local working branch `issue_56` to `2024_fall_uml`.
- Ensured the lines you had referenced were amended.
  - It appears they had already been removed in the `2024_fall_uml` branch.
- Adjusted my phrasing to match the context of the `2024_fall_uml` branch.
- Force pushed to my fork to reflect the rebase and changes.
- Updated target branch to be `2024_fall_uml`

**Review by charliemirabile — APPROVED** [review]
lgtm

**Pull request merged**